requirements_base.txt

```diff
@@ -121,3 +121,8 @@ wrapt==1.12.1             # via astroid
 xmltodict==0.12.0         # via -r requirements.in
 zope.event==4.4           # via gevent
 zope.interface==5.1.0     # via gevent
+# DA-3099 -- dchan
+Flask-GraphQL==2.0.0
+graphene==3.1.0
+graphql-core==3.1
+names
```

Comment:
Are any of these dependencies needed other than `graphene`? I think `graphene` brings in `graphql-core` on it's own, I'm not seeing where `flask-graphql` was used, and I think `names` was something from earlier testing, right?

Reply:
removed unused package in requirements_base.txt